src/macros.rs

```diff
@@ -0,0 +1,44 @@
+/// Implements conversion traits on a type wrapping a three-rs type. Useful for when you wrap a
+/// three-rs type with your own struct. Allows you to use that struct in place of any three-rs
+/// object.
+///
+/// Implements the following traits:
+///
+/// * `AsRef<NodePointer>`
+/// * `Deref<Target=Object>`
+/// * `DerefMut<Object>`
+///
+/// ```
```

Comment:
Replace this line with 
```
\`\`\`rust,no_run
```
to prevent running this as doc test (cause error during Travis CI)